src/EditCell.tsx

```diff
@@ -84,7 +83,12 @@ export default function EditCell<R, SR>({
       onMouseDownCapture={cancelFrameRequest}
     >
       {column.editor != null && (
```

Comment:
Is it possible to render an `EditCell` for a column with no editor? 🤔

Reply:
probably not. Let's revisit the types